Package.swift

```diff
@@ -0,0 +1,28 @@
+// swift-tools-version:5.2
```

Comment:
Can we also have standard license header added here?

Reply:
@rajdeep this comment needs to be on top, otherwise SPM doesn't work. I can add the license below it.